android/java/org/chromium/chrome/browser/crypto_wallet/fragments/PortfolioFragment.java

```diff
@@ -337,11 +339,26 @@ public void onAssetClick(BlockchainToken asset) {
         if (selectedNetwork == null) {
             return;
         }
+
         if (asset.isErc721 || asset.isNft) {
-            // TODO: show nft details of clicked nft
-            return;
+            PortfolioModel.NftDataModel selectedNft = null;
+            List<PortfolioModel.NftDataModel> nftModels = mPortfolioModel.mNftModels.getValue();
```

Comment:
Can't see https://github.com/brave/brave-core/pull/16671#discussion_r1072356407 change here. Might have missed to push?

Reply:
Sorry, the push didn't go through. Try now, please.

Reply:
Np, thanks, already approved.